plugins/txn_test_gen_plugin/txn_test_gen_plugin.cpp

```diff
@@ -368,10 +370,12 @@ struct txn_test_gen_plugin_impl {
          throw fc::exception(fc::invalid_operation_exception_code);
       timer.cancel();
       running = false;
+      auto elapsed_ms = (fc::time_point::now() - _start_time).count()/1000;
       ilog("Stopping transaction generation test");
 
       if (_txcount) {
-         ilog("${d} transactions executed, ${t}us / transaction", ("d", _txcount)("t", _total_us / (double)_txcount));
+         ilog("${d} transactions executed for ${s} ms (${tps} TPS), ${t} us / transaction",
+            ("d", _txcount) ("s", elapsed_ms) ("tps", (uint16_t)(((double)_txcount/elapsed_ms)*1000)) ("t", _total_us / (double)_txcount));
```

Comment:
"tps" here is number transactions submitted per second not executed. This is misleading and not what is typically reported as tps.